earth_enterprise/src/fusion/autoingest/JsUtils.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright 2017 Google Inc. Copyright 2018 the Open GEE Contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+#include "fusion/autoingest/JsUtils.h"
+#include <math.h>
```

Comment:
I think we really just need the includes for JsUtils.h and array here.

Reply:
Yes keep headers as light as you can.